lib/pos_mempool_test.go

```diff
@@ -226,7 +226,7 @@ func TestPosMempoolUpdateGlobalParams(t *testing.T) {
 	newGlobalParams := _testGetDefaultGlobalParams()
 	newGlobalParams.MinimumNetworkFeeNanosPerKB = 20000
 	mempool.UpdateGlobalParams(newGlobalParams)
-	require.Equal(0, len(mempool.GetTransactions()))
+	require.Equal(100, len(mempool.GetTransactions()))
```

Comment:
refreshNoLock no longer removes txns so the number of txns in the mempool should be unaffected